src/EC2Store.js

```diff
@@ -22,7 +22,59 @@ class EC2Store {
 			let ec2 = new AWS.EC2();
 			ec2.describeSnapshots({}, function (err, response) {
 				if (err) reject(err);
-				else resolve(response);
+
+				else {
+					var taggedForBackup = response.Snapshots.filter(function(obj){
```

Comment:
Be careful naming your variables. Calling things `obj` and `obj2` can make the code harder to read, especially when they are reused elsewhere. For example, this `obj` is a snapshot element in an array.